dirsrvtests/tests/suites/webui/monitoring/monitoring_test.py

```diff
@@ -227,6 +231,60 @@ def test_logging_visibility(topology_st, page, browser_name):
     assert frame.locator('#securitylog-area').is_visible()
 
 
+def test_create_credential_and_alias(topology_st, page, browser_name):
+    """ Test check that you are able to give input to input field in pop up windows when creating credential or alias
+
+    :id: 8908405c-47b9-470e-a906-42790b131e9f
+    :setup: Standalone instance
+    :steps:
+         1. Check if replication is enabled, if not enable it.
+         2. Click on Monitoring tab, click on Replication Log button on side panel.
+         3. Click on Add Credentials button and fill Hostname and Password, then click on save.
+         4. Check if new credential appeared in the credentials list.
+         5. Click on Add Alias button and fill alias name and alias hostname, click on Save button.
+         6. Check if new alias appeared in the alias list.
+    :expectedresults:
+         1. Success
+         2. Success
+         3. Success
+         4. Element is visible
+         5. Success
+         6. Element is visible
+    """
+    setup_login(page)
+    time.sleep(1)
+    frame = check_frame_assignment(page, browser_name)
+
+    log.info('Check if replication is enabled, if not enable it in order to proceed further with test.')
+    frame.get_by_role('tab', name='Replication').click()
+    time.sleep(2)
+    if frame.get_by_role('button', name='Enable Replication').is_visible():
+        frame.get_by_role('button', name='Enable Replication').click()
+        frame.fill('#enableBindPW', 'redhat')
+        frame.fill('#enableBindPWConfirm', 'redhat')
+        frame.get_by_role("dialog", name="Enable Replication").get_by_role("button",
+                                                                           name="Enable Replication").click()
+        frame.get_by_role('button', name='Add Replication Manager').wait_for()
+        assert frame.get_by_role('button', name='Add Replication Manager').is_visible()
```

Comment:
I wonder if it'll be possible (and it'll make sense) to put it as a fixture? It's already used in test_replication_visibility and can be used in other places.

Reply:
It could be placed into webui/__init.py for easier setup for other and future replication tests.
Then we could keep at least the assert line in the original test_replication_visibility. 
Or end it differently by testing the fixture itself?

Reply:
What do you think of Barbora's suggestion? @droideck

Reply:
Yep, sounds good!

Reply:
@droideck Is it OK like this?